mappings/net/minecraft/world/border/WorldBorder.mapping

```diff
@@ -94,3 +97,39 @@ CLASS net/minecraft/class_2784 net/minecraft/world/border/WorldBorder
 		METHOD <init> (Lnet/minecraft/class_2784;D)V
 			ARG 1 size
 		METHOD method_11996 recalculateBounds ()V
+	CLASS class_5200 Config
+		FIELD field_24123 centerX D
+		FIELD field_24124 centerZ D
+		FIELD field_24125 damagePerBlock D
+		FIELD field_24126 safeZone D
+		FIELD field_24127 warningBlocks I
+		FIELD field_24128 warningTime I
+		FIELD field_24129 size D
+		FIELD field_24130 sizeLerpTime J
+		FIELD field_24131 sizeLerpTarget D
```

Comment:
```suggestion
		FIELD field_24130 lerpTime J
		FIELD field_24131 lerpTarget D
```

Reply:
These are Mojang names. Lerp time and lerp target for the worldborder's size.